### PR TITLE
graphite2: needs GCC_RUNTIME_PKG

### PR DESCRIPTION
**Pull request by tsoome**

### CODE REVIEW

File: components/library/graphite2/Makefile [hunk 1]
```diff
@@ -62,6 +63,7 @@ install:	$(INSTALL_32_and_64)
 test:		$(TEST_32_and_64)
 
 # Auto-generated dependencies
-REQUIRED_PACKAGES += system/library
+REQUIRED_PACKAGES += $(GCC_RUNTIME_PKG)
```
Comment (AndWac) [inline components/library/graphite2/Makefile:67]: On my build env GCC_RUNTIME_PKG is not automatically detected by gmake REQUIRED_PACKAGES.
Our convention is to add additionally needed packages under a separate label, eg. # build dependency.

Comment (tsoome) [reply components/library/graphite2/Makefile:67]: like this?

Comment (AndWac) [reply components/library/graphite2/Makefile:67]: Almost, just forgot to add a line like
\# Manually added build dependencies
REQUIRED_PACKAGES += $(GCC_RUNTIME_PKG)

\# Auto-generated dependencies
...

Comment (AndWac) [reply components/library/graphite2/Makefile:65]: Sigh should have used preview.

Comment (pyhalov) [reply components/library/graphite2/Makefile:67]: So, it's detected on SPARC, but not on x86?

Comment (tsoome) [reply components/library/graphite2/Makefile:67]: yep.

Comment (pyhalov) [reply components/library/graphite2/Makefile:67]: Can you modify comment so that it describes this.

Comment (tsoome) [reply components/library/graphite2/Makefile:67]: done

**Pull request merged**